lib/liquid/combinators/tags/custom_block.ex

```diff
@@ -0,0 +1,129 @@
+defmodule Liquid.Combinators.Tags.Custom_Block do
```

Comment:
`Custom_Block` -> `CustomBlock`
`Custom_Tag` -> `CustomTag`
When you use `cameCase` or `CapitalCase` you didn't need split the word with a underscore. The capital letter is the separator between words.